packages/next/src/server/future/route-modules/app-page/vendored/contexts/amp-context.ts

```diff
@@ -0,0 +1,3 @@
+module.exports = require('../../module.compiled').vendored[
+  'contexts'
```

Comment:
I'm assuming this is to skip certain inlining / bundling? Otherwise it would be `.vendored.contexts`